### PR TITLE
chore(project): update release proced. acc. to recent process updates

### PR DESCRIPTION
**Pull request by MaxTru**
__Which issue does this PR address?__

addresses https://github.com/bpmn-io/internal-docs/issues/152

__Acceptance Criteria__

* [ ] Corresponds to the concept  https://github.com/bpmn-io/internal-docs/issues/152
* [ ] Corresponds to the design follows existing structure

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: docs/.project/PATCH_RELEASE.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
+* [alüha release documentation](./ALPHA_RELEASE.md), or
```
Comment (pinussilvestrus) [inline docs/.project/PATCH_RELEASE.md:4]: `alpha`?

alüha reminds me of aloha 🥥  🌴

Comment (MaxTru) [reply docs/.project/PATCH_RELEASE.md]: Fixed

File: docs/.project/RELEASE.md [hunk 2]
```diff
@@ -24,12 +26,13 @@ Note: The release manager in charge will be determined by the Camunda Modeler Re
 _To be done immediately after creating this issue._
 
 * [ ] put up code freeze appointment in calendar (include `modeling`, `qa`, `infra`, `zeebe` and `Team-Support`)
-* [ ] put up release appointment in calendar (include `modeling` and Marketing [Charley Mann & Christopher Rogers])
+* [ ] put up release appointment in calendar (include `modeling`, `DevRel` and Marketing [Charley Mann & Christopher Rogers])
 
 _To be done after code freeze and prior to the release day to prepare and build the release._
 
-* [ ] make sure dependencies are released (`rm -rf node_modules && npm i && npm run all` works)
+* [ ] make sure changes upstream libraries are merged and released
```
Comment (pinussilvestrus) [inline docs/.project/RELEASE.md:33]: changes _in_ upstream libraries

Comment (MaxTru) [reply docs/.project/RELEASE.md]: Fixed, will merge now

**Pull request merged**